dashboard/app/models/pd/application/application_base.rb

```diff
@@ -71,6 +71,19 @@ def set_type_and_year
     validates_presence_of :type
     validates_presence_of :status
 
+    # Override in derived class to provide headers
+    # @return [String] csv text row of column headers, ending in a newline
+    def self.csv_header
+      "\n"
```

Comment:
I suggest raising an exception here - we should expect every subclass to override this